SPECS/kmod-drbd.spec

```diff
@@ -97,8 +121,10 @@ make -C drbd install KDIR=/lib/modules/%{kernel_version}/build \
 mkdir -p %{buildroot}/etc/depmod.d
 find %{buildroot}/lib/modules/*/ -name "*.ko"  -printf "%%P\n" |
 sort | sed -ne 's,^extra/\(.*\)/\([^/]*\)\.ko$,\2 \1,p' |
-xargs -r printf "override %%-16s * weak-updates/%%s\n" \
-> %{buildroot}/etc/depmod.d/drbd.conf
+while read -r mod path; do
+	printf "override %%-16s * weak-updates/%%s\n" $mod $path
+	printf "override %%-16s %%s extra/%%s\n" $mod $kernelrelease $path
+done > %{buildroot}/etc/depmod.d/drbd.conf
```

Comment:
This command is quite complex!
How do we make sure it keeps doing what it's supposed to do?

Reply:
I'm sure it works as before since it's an upstream change from linbit. :)

Reply:
No, but for more details:
- The upstream DRBD specfile is directly in their github source repo; there is no SRPM repo.
- The XCP-ng spec file is close to the one used by linbit, but with modifications so that we can use it on our side (removal of unnecessary code, and especially fixes for some build incompatibilities).
- I try to be in sync with their spec file with each source update; this requires verifying that our coccinelle version is compatible, for example.